app/src/androidTest/java/org/mozilla/fenix/ui/robots/NavigationToolbarRobot.kt

```diff
@@ -98,12 +97,15 @@ class NavigationToolbarRobot {
             mDevice.pressEnter()
 
             runWithIdleRes(sessionLoadedIdlingResource) {
-                onView(
-                    anyOf(
-                        withResourceName("browserLayout"),
-                        withResourceName("download_button"),
-                    ),
-                ).check(matches(withEffectiveVisibility(ViewMatchers.Visibility.VISIBLE)))
+                assertTrue(
+                    mDevice.findObject(
+                        UiSelector().resourceId("$packageName:id/browserLayout"),
+                    ).waitForExists(waitingTime) || mDevice.findObject(
+                        UiSelector().resourceId("$packageName:id/download_button"),
+                    ).waitForExists(waitingTime) || mDevice.findObject(
+                        UiSelector().text(getStringResource(R.string.tcp_cfr_message)),
+                    ).waitForExists(waitingTime),
+                )
```

Comment:
What's the context on this change?

Reply:
When enabled, the TCP CFR message appears on top of the browser layout and will block the view so the test won't see it.
I wanted to keep the Espresso matchers but for some reason, they didn't match the text in the CFR.